plugins/modules/cloud/misc/proxmox.py

```diff
@@ -140,9 +140,10 @@
     type: str
   unprivileged:
     description:
-      - Indicate if the container should be unprivileged
+      - Indicate if the container should be unprivileged.
+      - The default value for this param is C(no) but that is being deprecated.
+      - It will be replaced with C(yes) in community.general 7.0.0
```

Comment:
Please use `true/false` instead of `yes/no` - I know there are docs out there saying we should use `yes/no`, that was a recommendation a while ago but everyone seems to be converging to `true/false` as the best practice.